src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,18 @@
+const CHECK_STATUS = 'categories/CHECK_STATUS';
+
+export const checkStatus = () => ({
+  type: CHECK_STATUS,
+});
+
+const initialState = 'Under construction';
+
+const categoriesReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case CHECK_STATUS:
+      return state;
+    default:
+      return state;
+  }
+};
```

Comment:
- [ ] Same here, Please use `createSlice` instead of using the old `redux` method here.